frontend/src/pages/pipelines/global/modelCustomization/trainingHardwareSection/TrainingHardwareProfileFormSection.tsx

```diff
@@ -81,9 +81,12 @@ const TrainingHardwareProfileFormSection: React.FC<TrainingHardwareProfileFormSe
                 <>
                   <Content component={ContentVariants.p}>
                     This list includes only hardware profiles with defined GPUs.
-                    {isProjectScoped &&
-                      projectScopedHardwareProfiles['1'] &&
-                      projectScopedHardwareProfiles['0'].length > 0 && (
+                  </Content>
+                  {isProjectScoped &&
+                    projectScopedHardwareProfiles['1'] &&
+                    projectScopedHardwareProfiles['0'].length > 0 && (
+                      <>
+                        <div style={{ marginTop: 10, marginBottom: 8 }} />
```

Comment:
We should avoid using these hardcoded value and the custom styles if possible. When meeting these layout issues, check the PatternFly doc to see if there are existing solutions. Here are some solutions you can consider
1. Replace `<List>` and `<ListItem>` with the `<Content>` as [the example](https://www.patternfly.org/components/content#unordered-list) shows, that could generate a proper space between the element above and below
2. Consider using [Stack layout](https://www.patternfly.org/layouts/stack) here. This could ensure every element in it has some specific spaces

I'd recommend the first solution because we should try to follow what PatternFly designs as much as possible, that could make sure the layout to follow their design guidelines and standard.